jetty-websocket/javax-websocket-server/src/main/java/org/eclipse/jetty/websocket/javax/server/JavaxWebSocketServerContainer.java

```diff
@@ -308,18 +337,18 @@ public void setAsyncSendTimeout(long ms)
     @Override
     public void setDefaultMaxBinaryMessageBufferSize(int max)
     {
-        this._webSocketCreatorMapping.setDefaultMaxBinaryMessageSize(max);
+        customizer.setMaxBinaryMessageSize(max);
```

Comment:
Huh?
Customizer does what exactly?

Reply:
The customizer has all the fields that can be set on a session (see `FrameHandler.Configuration`) and knows if they have been set or are defaulted.  WebSockets sessions are now configured by applying all the appropriate configurations.  The container itself may have some configurations in its customizer that apply to all contained websockets.  An individual mapping may have a customizer applied to just that mapping.   Individual upgrade filters/servlets may also have customizers so you can apply configuration via init params that affect just some websockets.